.github/workflows/ci.yml

```diff
@@ -53,10 +53,11 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
+        # We test on all LTS, maintenance and current versions of node above 22.X
         node:
+          - version: 24.x
           - version: 23.x
```

Comment:
I think we can remove 23.x if 24 is out: it's non-LTS, so it won't receive updates.

Reply:
why not keep maintenance ?

Reply:
We could i suppose. I think people are not supposed to use dead lines though, so we "shouldn't" to incentivise people to not. If you really want, it's fine 🙂

Reply:
Whichever you prefer 😉 just resolve this thread if you wanna keep it